source/includes/changelogs/releases/100.7.3.rst

```diff
@@ -0,0 +1,22 @@
+.. _100.7.3-changelog:
```

Comment:
We'll also need to update the snooty.toml release version.

Reply:
Fixed